Corona-Warn-App/src/main/java/de/rki/coronawarnapp/familytest/core/storage/FamilyTestDatabase.kt

```diff
@@ -96,10 +102,16 @@ interface FamilyCoronaTestDao {
     suspend fun get(identifier: TestIdentifier): FamilyCoronaTestEntity?
 
     @Transaction
-    suspend fun update(identifier: TestIdentifier, update: (FamilyCoronaTest) -> FamilyCoronaTest) {
+    suspend fun update(identifier: TestIdentifier, update: suspend (FamilyCoronaTest) -> FamilyCoronaTest) {
         get(identifier)?.let {
             val updated = update(it.test).toEntity()
             insert(updated)
         }
     }
+
+    @Transaction
+    suspend fun updateAll(identifiers: Set<TestIdentifier>, update: suspend (FamilyCoronaTest) -> FamilyCoronaTest) {
```

Comment:
What about returning all items and let the caller fiter them, if needed?

Reply:
what do you mean , by returning all items? this one just writes to database the updates ,should anything be returned
?